### PR TITLE
Temporarily disable f16 xdlop E2E tests

### PR DESCRIPTION
**Pull request by yiqian1**
Disable E2E tests with small dimensions that failed in nightly CI on gfx908.

### CODE REVIEW

**Comment by whchung** [comment]
@yiqian1 That's quite a lot of tests being disabled. Please explain the symptoms why they fail.

Also please raise a ticket to track them.

**Review by whchung — CHANGES_REQUESTED** [review]
There are too many disabled on this PR. Let's also raise a ticket to track them.

I'll spend some time looking into this today 5/11/2021.

File: mlir/test/auto_e2e/conv2d_host_validation_f16_fwd.mlir [hunk 1]
```diff
@@ -73,31 +73,31 @@
...
 
 // CHECK_FILTER1: Unranked Memref base@ = 0x{{.*}} rank = 1 offset = 0 sizes = [1] strides = [1] data =
 // CHECK_FILTER1: [1]
 
 // COM: filter 1x1
-// RUN: mlir-miopen-driver -pv %random_data %xdlops -fil_layout=kcyx -in_layout=nchw -out_layout=nkhw  -batchsize=64 -in_channels=4 -out_channels=64 -in_h=32 -in_w=32 -fil_h=1 -fil_w=1   --dilation_h=1 --dilation_w=1 --padding_h=0 --padding_w=0 --conv_stride_h=1 --conv_stride_w=1 -p=false -t f16 -c | mlir-rocm-runner --shared-libs=%rocm_wrapper_library_dir/librocm-runtime-wrappers%shlibext,%linalg_test_lib_dir/libmlir_runner_utils%shlibext --entry-point-result=void | FileCheck %s --check-prefix=CHECK_FILTER2
+// FIXME: mlir-miopen-driver -pv %random_data %xdlops -fil_layout=kcyx -in_layout=nchw -out_layout=nkhw  -batchsize=64 -in_channels=4 -out_channels=64 -in_h=32 -in_w=32 -fil_h=1 -fil_w=1   --dilation_h=1 --dilation_w=1 --padding_h=0 --padding_w=0 --conv_stride_h=1 --conv_stride_w=1 -p=false -t f16 -c | mlir-rocm-runner --shared-libs=%rocm_wrapper_library_dir/librocm-runtime-wrappers%shlibext,%linalg_test_lib_dir/libmlir_runner_utils%shlibext --entry-point-result=void | FileCheck %s --check-prefix=CHECK_FILTER2
```
Comment (whchung) [inline mlir/test/auto_e2e/conv2d_host_validation_f16_fwd.mlir]: @yiqian1 from the look for your very brief description without knowing the details, is it possible enabling `%xdlops` cause the error, so they fail ONLY when we enable `-x2`?

in such case, how about we remove `%xdlops` for these configs?

**Comment by whchung** [comment]
@yiqian1 Also, do these configs fail if we revert #180 ?

File: mlir/test/auto_e2e/conv2d_host_validation_f16_fwd.mlir (hunk 1 above)
Comment (yiqian1) [reply mlir/test/auto_e2e/conv2d_host_validation_f16_fwd.mlir]: Yes. I think I can remover -x2.

Comment (whchung) [reply mlir/test/auto_e2e/conv2d_host_validation_f16_fwd.mlir]: Please confirm / deny my hypothesis. Are they failing ONLY when we enable `-x2`?

Comment (whchung) [reply mlir/test/auto_e2e/conv2d_host_validation_f16_fwd.mlir]: ping? please help revise the description of the PR if that's the case.

**Comment by yiqian1** [comment]
No, reverting #180 doesn't help. I think the problem existed before #180 was in.

**Review by whchung — APPROVED** [review]

**Pull request merged**